openedx/features/student_certificates/views.py

```diff
@@ -111,14 +113,28 @@ def student_certificates(request):
             course_title = course.display_name
 
         user_certificates.append({
-            'course_name': course_name,
-            'course_title': course_title,
+            'certificate_name': course_name,
+            'certificate_title': course_title,
             'social_sharing_urls': get_philu_certificate_social_context(course, certificate),
             'certificate_url': "%s%s" % (settings.LMS_ROOT_URL, certificate_url),
             'course_start': start_date.strftime('%b %d, %Y') if start_date else None,
             'completion_date': completion_date.strftime('%b %d, %Y') if completion_date else None,
+            'is_program_cert': False,
         })
 
+    program_credentials = get_credentials(request.user, credential_type='program')
```

Comment:
It would be great if you write another method to get credential_certificates and move this new block in that method

Reply:
and call this method before following line, So that programs will be displayed first on my certificates page
available_certificates = GeneratedCertificate.objects.filter(user=user, course_id__in=enrolled_course_ids).all()

Reply:
and it would be great if you do the same for course certificates